Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -529,14 +549,36 @@ class BocList
 
   private static FixValidationErrorOverflowSize(scrollableContainer: HTMLElement)
   {
-    // Sets the width of the validation error field to be as wide as the visible part of the bocList.
+    // Sets the width of the validation error field to be as wide as the visible part of the bocList
+    // minus how much space the td on the left requires to align with the error indicator column.
     // This enables sticky behaviour while still allowing for a scrollbar if required.
 
-    const width = LayoutUtility.GetWidth(scrollableContainer);
+    const availableWidthInContainer = scrollableContainer.clientWidth;
+
+    const td: Nullable<HTMLElement> = scrollableContainer.querySelector('tr.bocListValidationRow td:first-child:not(.bocListValidationFailureCell)');
+
+    let widthBeforeValidationFailureColumn = 0;
+    // width does not matter if the first td is already the one with the error messages (and therefore has child nodes)
+    if (td != null && !td.hasChildNodes())
+    {
+      widthBeforeValidationFailureColumn = Math.max(td.clientWidth - scrollableContainer.scrollLeft, 0);
+    }
+
+    const overflowContainers = [...scrollableContainer.querySelectorAll<HTMLElement>(':scope tr.bocListValidationRow > td.bocListValidationFailureCell > div')];
+
+    overflowContainers.forEach(container => {
 
-    const overflowContainers = [...scrollableContainer.querySelectorAll<HTMLElement>(':scope tr.bocListValidationRow > td > div')];
-    overflowContainers.forEach(e => {
-      e.style.width = "calc(" + width.toString() + "px - var(--remotion-themed-scrollbar-width))";
+      let padding = 0;
+      let border = 0;
+
+      let parentTd = container.parentNode as HTMLElement;
+      if(parentTd != null){
+        let style = window.getComputedStyle(parentTd);
+
+        padding = parseFloat(style.paddingLeft) + parseFloat(style.paddingRight);
+        border = parseFloat(style.borderLeftWidth) + parseFloat(style.borderRightWidth);
+      }
```

Comment:
```suggestion
      let style = window.getComputedStyle(container.parentNode!);
      padding = parseFloat(style.paddingLeft) + parseFloat(style.paddingRight);
      border = parseFloat(style.borderLeftWidth) + parseFloat(style.borderRightWidth);
```

The container.parentNode can't be null at this point because we just got it from the page. You may need to add a cast back in, just not a soft-cast if possible. That way, we can simplify a lot and don't need to invent names like `parentTd`.